cdc/model/http_model.go

```diff
@@ -25,12 +25,24 @@ import (
 // JSONTime used to wrap time into json format
 type JSONTime time.Time
 
-// MarshalJSON use to specify the time format
+// MarshalJSON used to specify the time format
 func (t JSONTime) MarshalJSON() ([]byte, error) {
-	stamp := fmt.Sprintf("\"%s\"", time.Time(t).Format("2006-01-02 15:04:05.000"))
+	stamp := fmt.Sprintf(`"%s"`, time.Time(t).Format("2006-01-02 15:04:05.000"))
```

Comment:
may be we can extract `"2006-01-02 15:04:05.000"` as a const like
```go
timeFormat = `"2006-01-02 15:04:05.000"` 
```

Reply:
> may be we can extract `"2006-01-02 15:04:05.000"` as a const like
> 
> ```go
> timeFormat = `"2006-01-02 15:04:05.000"` 
> ```

rest LGTM

Reply:
> may be we can extract `"2006-01-02 15:04:05.000"` as a const like
> 
> ```go
> timeFormat = `"2006-01-02 15:04:05.000"` 
> ```

modified in latest commit.